File: src/codes/Users_Manual/gain/focus_1.cpp
//~#include<autd3.hpp>
//~int main() {
//~const auto x = 0.0;
//~const auto y = 0.0;
//~const auto z = 0.0;
const auto g =
    autd3::gain::Focus(autd3::Vector3(x, y, z))
        .with_intensity(std::numeric_limits<autd3::EmitIntensity>::max())
        .with_phase_offset(autd3::Phase(0x00));
//~return 0; }

File: src/codes/Users_Manual/gain/focus_1.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Utils;
~using AUTD3Sharp.Gain;
~var x = 0.0;
~var y = 0.0;
~var z = 0.0;
var g = new Focus(new Vector3d(x, y, z))
            .WithIntensity(EmitIntensity.Max)
            .WithPhaseOffset(new Phase(0x00));